code/style.css

```diff
@@ -1,4 +1,260 @@
-html {
-  background: #0025ff;
+body {
+  background:#F2E8C4;
+  font-family: 'balsamiq Sans', cursive;
   color: #fff;
+  margin: 0;
+  padding: 0;
 }
+
+h1 {
```

Comment:
What do you think about decreasing the font-size a bit in mobile? :)

<img width="444" alt="Skärmavbild 2021-02-02 kl  16 12 01" src="https://user-images.githubusercontent.com/51529201/106619955-65e2bc00-6571-11eb-9957-e800752cfc5a.png">